src/client/systems/sysstarsystemrenderer.cpp

```diff
@@ -205,12 +198,13 @@ void conquerspace::client::systems::SysStarSystemRenderer::Render() {
     glDepthFunc(GL_LESS);
     skybox_renderer.EndDraw();
 
-    planet_renderer.RenderBuffer();
```

Comment:
This was put in a specific order. I wanted the planets to be in front of the billboards because I wanted planets to cover them, so it was easier to see the planet surfaces. If you have any arguments for doing this, feel free to convince me.